galeracluster/source/training/tutorials/starting-cluster.rst

```diff
@@ -92,31 +92,31 @@ Once you have at least three hosts ready, you can initialize the cluster.
 
 By default, a node don't start as part of the :term:`Primary Component`.  Instead, it assumes that the Primary Component is already running and it is merely joining an existing cluster.  For each node it encounters in the cluster, it checks whether or not it's a part of the Primary Component.  When it finds the Primary Component, it requests a state transfer to bring its database into sync with the cluster.  If it can't find the Primary Component, it will remains in a non-operational state.
 
-The problem is that there is no Primary Component when a cluster starts, when the first node is initiated.  Therefore, you need explicitly to tell that first node to do so with the ``--wsrep-new-cluster`` argument.  Althought this initiate node is said to be the first node, it can fall behind and leave the cluster without necessarily affecting the Primary Component.
+The problem is that there is no Primary Component when a cluster starts, when the first node is initiated.  Therefore, you need explicitly to tell that first node to do so with the ``mysqld_bootstrap`` command.  Althought this initiate node is said to be the first node, it can fall behind and leave the cluster without necessarily affecting the Primary Component.
 
 .. note:: When you start a new cluster, any node can serve as the first node, since all the databases are empty.  When you migrate from MySQL to Galera Cluster, use the original master node as the first node.  When restarting the cluster, use the most advanced node.  For more information, see :doc:`Migration <./migration>` and :doc:`Quorum Reset <../../../documentation/quorum-reset>`.
 
-To start the first node--which should have MySQL, MariaDB or Percona XtraDB Cluster, and Galera installed--you'll have to launch the database server on it with the ``--wsrep-new-cluster`` option.  There are different ways to do this, depending on the operating system. For systems that use ``init``, execute the following from the command-line:
+To start the first node--which should have MySQL, MariaDB or Percona XtraDB Cluster, and Galera installed--you'll have to launch the database server on it with the ``mysqld_bootstrap`` command.  There are different ways to do this, depending on the operating system. For systems that use ``init``, execute the following from the command-line:
 
 On Galera:
 
 .. code-block:: console
 
-   mysqld_bootstrap --wsrep-new-cluster
+   mysqld_bootstrap
 
 On MariaDB:
 
 .. code-block:: console
 
-   mysqld --wsrep-new-cluster
+   mysqld mysqld_bootstrap
```

Comment:
No. 

Technically you can use galera_new_cluster

ref: https://mariadb.com/kb/en/getting-started-with-mariadb-galera-cluster/

Reply:
Back to original